go.mod

```diff
@@ -53,6 +53,7 @@ require (
 	github.com/hashicorp/terraform-exec v0.17.3
 	github.com/jongio/azidext/go/azidext v0.5.0
 	github.com/kdomanski/iso9660 v0.2.1
+	github.com/libvirt/libvirt-go-xml v5.10.0+incompatible
```

Comment:
If this change copies large chunks of the provider code, it makes sense to do the libvirtxml migration separately, so that we don't block the whole effort on it.